app/models/user.rb

```diff
@@ -5,7 +5,7 @@ class User < ActiveRecord::Base
   has_many :favorites, dependent: :destroy
   before_save { self.email = email.downcase if email.present? }
   before_save { self.role ||= :member }
-  before_create :generate_auth_token
+  # before_create :generate_auth_token
```

Comment:
Not sure where this came from?  Couldn't find it in any of the checkpoints.  Let's comment it out for now and reintroduce if we run into issues with it later.